docs/conf.py

```diff
@@ -55,6 +55,9 @@
     r"https://doi\.org/.*",
     r"https://link\.aps\.org/doi/.*",
     r"http://dx\.doi\.org/.*",
+    r"https://www\.quantiki\.org/wiki/entanglement-formation",
+    r"https://www\.quantiki\.org/wiki/trace-distance",
+    r"https://www\.quantiki\.org/wiki/trace-norm",
```

Comment:
Is there a reason we are adding these slug domains specifically? Why not just regex the whole domain? That is `https://www\.quantiki\.*`?

Additionally, I don't think we want to ignore this, we want to make sure that we hit this domain!